core/services/ocr2/plugins/ocr2keeper/evm21/logprovider/abi.go

```diff
@@ -0,0 +1,39 @@
+package logprovider
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/logpoller"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/i_log_automation"
+)
+
+type LogDataPacker interface {
+	PackLogData(log logpoller.Log) ([]byte, error)
+}
+
+type logEventsPacker struct {
+	abi abi.ABI
+}
+
+func NewLogEventsPacker(logDataABI abi.ABI) *logEventsPacker {
+	return &logEventsPacker{abi: logDataABI}
+}
+
+func (p *logEventsPacker) PackLogData(log logpoller.Log) ([]byte, error) {
+	topics := [][32]byte{}
+	for _, topic := range log.GetTopics() {
+		topics = append(topics, topic)
+	}
+	return p.abi.Pack("checkLog", &i_log_automation.Log{
```

Comment:
should we include changes made [here](https://github.com/smartcontractkit/chainlink/pull/9669/files#diff-ef607d85bf697f41e3a57955f40f0bf09c82e94ef1a29d454cd9a18b2d499f26R52)?